src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,80 @@
+package core.basesyntax.service;
+
+import core.basesyntax.exception.UserValidationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final int VALID_AGE = 20;
+    private static final int UNDER_AGE = 17;
+    private static final String VALID_LOGIN = "validLogin";
+    private static final String VALID_PASSWORD = "validPassword";
+    private static final String SHORT_INPUT = "short";
+    private User user;
+    private final RegistrationService registrationService = new RegistrationServiceImpl();
+
+    @BeforeEach
+    void createValidUser() {
+        user = new User();
+        user.setLogin(VALID_LOGIN);
+        user.setPassword(VALID_PASSWORD);
+        user.setAge(VALID_AGE);
+    }
+
+    @Test
+    void register_nullUser_notOk() {
+        Assertions.assertThrows(UserValidationException.class, () -> {
```

Comment:
use static import

Reply:
Import statement for 'org.junit.jupiter.api.Assertions.assertEquals' is in the wrong order. Should be in the 'STATIC' group, expecting not assigned imports on this line. [CustomImportOrder]
Can't fix this message and can't move static imports before non static